### PR TITLE
README updates: add instructions for project setup to tutorial

### PR DESCRIPTION
**Pull request by metame**
Adds instructions for pinning the package from github in opam and clarifies how to run the app via dune.

### CODE REVIEW

**Comment by leostera** [comment]
Thanks for the contrib @metame! ✨ I'll leave a few comments 🙏🏼

File: README.md [hunk 1]
```diff
@@ -31,7 +31,21 @@ We'll start by defining our `dune` file:
...
 ```
 
-Great, now we can create a new `shop.ml` file and start by openig up `Minttea`:
+Then we need to pin the `minttea` package to the github source:
+
+`$ opam pin add minttea git@github.com:leostera/minttea.git --yes`
+
+Opam will do some work installing minttea from the github source.
+
+Next, we'll define our `dune-project` file:
```
Comment (leostera) [inline README.md:40]: I think we can tell folks to start the tutorial by running `dune init proj shop_tui`, so we can skip the manual creation of the dune files, and just adding the dependencies. Wdyt?

Comment (metame) [reply README.md:40]: I went back and forth with this as I was "testing it out". Here's what I came up with as options:

1. `dune init proj shop_tui` - This creates everything we need but of course also creates a lot that we don't need. It also could be a little confusing having the `lib`, for example, which remains completely unused and still referenced in the dune file. Of course, the target users are those familiar with OCaml so perhaps I'm overthinking this.
2. `dune init exec shop_tui` - This creates a `shop_tui.ml` file and a `dune` file. We'd still need to tell users to create the `dune-project` file as it's not created in this step. This is perhaps simpler but just felt clunky that we still had to fall back to manual creation of the `dune-project`.
3. Manual file creation (the current iteration) - This is fairly straightforward and doesn't create the noise in option 1 and eliminates what felt clunky in option 2, which is why I ended up going with this option. 

And, of course, the fun part is that `dune init proj` creates a directory while `dune init exec` doesn't so there's that difference in these options as well 😅. I see benefits in options 1 and 3, not so much option 2, but no strongly held opinions. 🙂

Comment (leostera) [reply README.md:40]: Makes sense, LGTM! 🚀

**Comment by leostera** [comment]
Thanks @metame 🤩

**Pull request merged**